src/app/view/components/dot-navigation/dot-navigation.component.spec.ts

```diff
@@ -216,6 +197,17 @@ describe('DotNavigationComponent', () => {
                 fixture.detectChanges();
             });
 
+            it('should expand menu', () => {
+                const firstItem: DebugElement = de.query(By.css('.dot-nav__list-item'));
+                expect(
+                    firstItem.nativeElement.classList.contains('dot-nav__list-item--active')
+                ).toBe(true);
+                const firstMenuLink: DebugElement = firstItem.query(By.css('.dot-nav-sub__link'));
+                expect(
+                    firstMenuLink.nativeElement.classList.contains('dot-nav-sub__link--active')
+                ).toBe(false);
+            });
+
```

Comment:
Don't we need a test to check if the routing is happening?

Reply:
that is tested already on line 185 when is collapsed and 211 when is expanded.